requirements.txt

```diff
@@ -4,3 +4,6 @@ docutils >= 0.16
 
 # For RSS
 feedgen >= 0.9.0  # For RSS feed
+
+# For catching typos
+codespell
```

Comment:
```suggestion
```

Not needed anymore, since `codespell` isn't installed directly and is instead only installed on-demand when needed by pre-commit